src/v/storage/log_replayer.cc

```diff
@@ -16,20 +16,13 @@
 #include "storage/logger.h"
 #include "storage/parser.h"
 #include "utils/vint.h"
+#include "bytes/utils.h"
 #include "vlog.h"
 
 #include <limits>
 #include <type_traits>
 
 namespace storage {
-static inline void crc_extend_iobuf(crc32& crc, const iobuf& buf) {
```

Comment:
unused code?

Reply:
no - it was identical to what's present in `bytes/utils.h` and I included it instead of this snippet